src/main/java/ladder/domain/Height.java

```diff
@@ -0,0 +1,38 @@
+package ladder.domain;
+
+import java.util.Objects;
+
+public class Height {
+    private static final String HEIGHT_ERROR_MESSAGE = "error : 높이는 최소%d, 최대%d 까지 가능합니다.";
+    private static final int MIN = 2;
+    private static final int MAX = 10;
+    private final int height;
```

Comment:
개인적으로는 상수와 일반 변수는 개행을 통해 구분해주면 좀 더 가독성이 좋다고 생각합니다 😄 
```suggestion
public class Height {
    private static final String HEIGHT_ERROR_MESSAGE = "error : 높이는 최소%d, 최대%d 까지 가능합니다.";
    private static final int MIN = 2;
    private static final int MAX = 10;
    
    private final int height;
```